flexget/components/emby/api_emby.py

```diff
@@ -1240,7 +1251,7 @@ def search(**kwargs):
         EmbyApi.set_provideres_search_arg(args, **kwargs)
         EmbyApi.set_common_search_arg(args)
 
-        args['SearchTerm'] = kwargs.get('title')
+        args['SearchTerm'] = EmbyApi.strip_year(kwargs.get('title'))
```

Comment:
I don't think you need a helper function to call the helper function. The existing split_title_year already does everything your helper does.
```suggestion
        args['SearchTerm'], _ = split_title_year(kwargs.get('title')
```
or
```suggestion
        args['SearchTerm'] = split_title_year(kwargs.get('title')[0]
```
Now that I've written that, I sorta want to change our helper to return a named tuple, so we could use `.title` rather than `[0]`

Reply:
I made that tool return a namedtuple now, so you can call `.title` on the result

Reply:
I pushed a new change, to remove the declaration of the now unused method

Reply:
I added the use of the named tuple